### ISSUE TITLE
Invalid path to mediainfo in artworkprocessor.py results in exception

### ISSUE THREAD

**Issue by b-jesch** [rmrector/script.artwork.dump#11]
```2022-03-15 15:10:53.043 T:9172    ERROR <general>: EXCEPTION Thrown (PythonToCppException) : -->Python callback/script returned the following error<--
                                                    - NOTE: IGNORING THIS CAN LEAD TO MEMORY LEAKS!
                                                   Error Type: <class 'ModuleNotFoundError'>
                                                   Error Contents: No module named 'python'
                                                   Traceback (most recent call last):
                                                     File "/home/jesch/.kodi/addons/script.artwork.dump/python/service.py", line 4, in <module>
                                                       from artworkprocessor import ArtworkProcessor
                                                     File "/home/jesch/.kodi/addons/script.artwork.dump/python/artworkprocessor.py", line 4, in <module>
                                                       from python.libs.mediainfo import MediaItem
                                                   ModuleNotFoundError: No module named 'python'
                                                   -->End of Python script error report<--```

**Comment by Ronny-nerd** [comment]
Have the same problem with Artwork Dump version 0.2.4

**Comment by nizo11** [comment]
I confirm this error.

**Comment by rob88v** [comment]
> 

I have this error as well.

**Comment by rmrector** [comment]
Fixed in 0.2.5

**Comment by Ronny-nerd** [comment]
Thanks rmrector for the quick fix with the new version 0.2.5 the bug was fixed again.
